content/deploy/README.md

```diff
@@ -35,9 +42,127 @@ Events:
   Normal  synchronize  3s         naiserator  successfully synchronized application resources (hash = 13485216922060251669)
 ```
 
-## Automated deploy
+## Automated deploy using Github
+This section describes how to use _NAIS deploy_ in order to deploy directly
+from cloud services such as CircleCI or TravisCI. You can use NAIS deploy also
+from Jenkins running in any internal zone. NAIS deploy is the canonical and
+preferred way to perform deployments on the NAIS platform.
 
-To automate the deployment, the nais.yaml manifest and `kubectl` must be available in your CI/CD pipeline.
+## How it works
+1. As the final step in one of your CI pipelines, a [deployment request](https://developer.github.com/v3/repos/deployments/#create-a-deployment) is created using GitHub's API. This will trigger a webhook we have set up at Github.
```

Comment:
```suggestion
1. When performing a deployment in your CI pipeline, create a [deployment request](https://developer.github.com/v3/repos/deployments/#create-a-deployment) is created using GitHub's API. This will trigger a webhook we have set up at Github.
```